app/src/main/java/nu/yona/app/security/EncryptionUtils.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2018 Stichting Yona Foundation
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+
+package nu.yona.app.security;
+
+import android.content.Context;
+import android.os.Build;
+
+import java.io.IOException;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.CertificateException;
+
+import nu.yona.app.utils.AppUtils;
+import nu.yona.app.utils.YonaRuntimeException;
+
+
+public class EncryptionUtils
+{
+
+	public static String encrypt(Context context, String plainText)
```

Comment:
Sorry for the nitpicking, but I asked for ``plaintext``, not ``plainText``. In cryptography, it's treated as a single word. See [Wikipedia](https://en.wikipedia.org/wiki/Plaintext).